packages/sanity/src/structure/i18n/resources.ts

```diff
@@ -139,6 +139,10 @@ const structureLocaleStrings = defineLocalesResources('structure', {
   /** The text for the banner that appears when a document is not in the current global release */
   'banners.release.not-in-release': 'Not in the <Label>{{title}}</Label> release.',
 
+  /** The text content for the unpublished document banner when is part of a release */
+  'banners.unpublished-release-banner.text':
+    'This document will be unpublished as part of the <strong>{{title}}</strong> release',
```

Comment:
```suggestion
    'This document will be unpublished as part of the <Label>{{title}}</Label> release',
```

Just to keep it consistent I see we use `Label` rather than `strong` for `banners.release.not-in-release`